### DISCUSSION TITLE
Plans for Privacy Manifests support

### DISCUSSION THREAD

**Discussion by tyomashin** [pointfreeco/swift-composable-architecture#2559] [Q&A] [answered]
Privacy Manifests were announced by Apple at WWDC 2023.
I am not sure if all libraries need this support,
Does TCA plan to support this?

* [Get started with privacy manifests - WWDC23 - Videos - Apple Developer](https://developer.apple.com/videos/play/wwdc2023/10060/)
* [Privacy manifest files | Apple Developer Documentation](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files)

Thanks again for all the great features you provide.

**Comment by mbrandonw** [comment, marked as answer]
Hi @tyomashin, we're more than willing to add this information to the library, but I couldn't find anything about how a purely SPM package can do it. Feel free to open a PR if you figure it out!

**Reply by ylorn** [reply]
Like this? https://github.com/SDWebImage/SDWebImage/blob/master/Package.swift#L35

**Reply by stephencelis** [reply]
@ylorn I believe privacy manifests are only required for packages that collect user data or hit certain APIs. TCA does not collect any user data, so I do not believe it needs to ship a privacy manifest. Please let us know if its lack of privacy manifest ever gets in the way of app submission, though. We'd be happy to talk to Apple if there is an issue.

**Reply by ylorn** [reply]
Thanks, glad to hear that. Just that widely-used APIs like `UserDefaults` are required to provide the manifest, TCA may not have used any of those, but not sure about other packages from pointfree are impacted. And the isowords app may need to embed that as well.
